modules/store/spec/store.spec.ts

```diff
@@ -471,4 +476,174 @@ describe('ngRx Store', () => {
       mockStore.dispatch(action);
     });
   });
+
+  describe('Meta Reducers', () => {
+    let metaReducerContainer: any;
+
+    beforeEach(() => {
+      metaReducerContainer = (function() {
+        function metaReducer1(reducer: ActionReducer<any, any>) {
+          return function(state: any, action: Action) {
+            return reducer(state, action);
+          };
+        }
+
+        function metaReducer2(reducer: ActionReducer<any, any>) {
+          return function(state: any, action: Action) {
+            return reducer(state, action);
+          };
+        }
+
+        return {
+          metaReducer1: metaReducer1,
+          metaReducer2: metaReducer2,
+        };
+      })();
+    });
+
+    it('should create a meta reducer for root and call it through', () => {
+      const metaReducerSpy: any = spyOn(
```

Comment:
Type this and the others to `: Spy`

Reply:
Could we also move this creation of these spies to inside the `beforeEach` since we need them in every test